storage/nox.py

```diff
@@ -71,10 +71,11 @@ def system_tests(session, python_version):
     # virutalenv's dist-packages.
     session.install('mock', 'pytest', *LOCAL_DEPS)
     session.install('../test_utils/')
-    session.install('.')
+    session.install('../pubsub')
+    session.install('-e', '.')
```

Comment:
@tseaver I seem to recall @lukesneeringer had a reason for not making system tests editable.

Reply:
Having the environments use the checked-out files (rather than the installed ones) is more convenient for debugging, I think, but it isn't essential to me.  I'll back it out if @lukesneeringer says so (I'd like to know why, of course).